docs/pkglists/CentOS_6.x86_64

```diff
@@ -13,19 +13,19 @@ x86_64/bareos-database-sqlite3
 x86_64/bareos-database-tools
 x86_64/bareos-devel
 x86_64/bareos-director
-x86_64/bareos-director-python-plugin
+x86_64/bareos-director-python2-plugin
 x86_64/bareos-filedaemon
```

Comment:
are the common packages missing? i.e. `bareos-director-python-plugins-common` from the build?